tests/README.md

```diff
@@ -78,3 +120,4 @@ docker run \
         --deploy-testing-infra \
         --path-to-testing-infra-manifests=data/manifests
 ```
+
```

Comment:
needed ?
just \n on the last non empty line is needed

Reply:
Done.